### ISSUE TITLE
[Bug]: Game Crashes After alt tabbing for 2 mins

### ISSUE THREAD

**Issue by fludar** [Yimura/YimMenu#1028]
### Describe the bug

I literally alt tabbed for 2 mins to see a tutorial then I got my game frozen and i can't do anything

### Steps To Reproduce

alt tab for 2 mins

### Expected Behavior

nothing.

### Screenshots and/or Logs

```
[18:35:28.1102824][WARN/notify.hpp:24] Blocked recursive infinite ped attachment crash from Daguy_ (24918032)
[18:38:19.5724860][ERROR/exception_handler.cpp:27] EXCEPTION_ACCESS_VIOLATION
Dumping modules:
GTA5.exe Base Address: 0x7FF73D080000 Size: 63296000
ntdll.dll Base Address: 0x7FF953B90000 Size: 1245184
KERNEL32.DLL Base Address: 0x7FF9523F0000 Size: 524288
KERNELBASE.dll Base Address: 0x7FF951480000 Size: 1613824
USER32.dll Base Address: 0x7FF952240000 Size: 606208
win32u.dll Base Address: 0x7FF9512E0000 Size: 49152
GDI32.dll Base Address: 0x7FF951D40000 Size: 57344
gdi32full.dll Base Address: 0x7FF9510D0000 Size: 655360
msvcp_win.dll Base Address: 0x7FF951820000 Size: 335872
ucrtbase.dll Base Address: 0x7FF950FB0000 Size: 798720
PSAPI.DLL Base Address: 0x7FF951E20000 Size: 4096
WS2_32.dll Base Address: 0x7FF953AD0000 Size: 290816
VERSION.dll Base Address: 0x7FF949C00000 Size: 12288
RPCRT4.dll Base Address: 0x7FF951A10000 Size: 876544
DSOUND.dll Base Address: 0x7FF90F030000 Size: 491520
msvcrt.dll Base Address: 0x7FF9520D0000 Size: 507904
CRYPT32.dll Base Address: 0x7FF951310000 Size: 1134592
advapi32.dll Base Address: 0x7FF951D70000 Size: 425984
WINTRUST.dll Base Address: 0x7FF9511F0000 Size: 307200
sechost.dll Base Address: 0x7FF952180000 Size: 446464
IMM32.dll Base Address: 0x7FF9519D0000 Size: 131072
SHLWAPI.dll Base Address: 0x7FF9533C0000 Size: 212992
combase.dll Base Address: 0x7FF953030000 Size: 2543616
SHELL32.dll Base Address: 0x7FF952840000 Size: 6488064
WININET.dll Base Address: 0x7FF92A510000 Size: 2113536
ole32.dll Base Address: 0x7FF9526A0000 Size: 897024
OLEAUT32.dll Base Address: 0x7FF951C50000 Size: 647168
WINMM.dll Base Address: 0x7FF946DB0000 Size: 114688
MF.dll Base Address: 0x7FF937040000 Size: 368640
MFPlat.DLL Base Address: 0x7FF930230000 Size: 1503232
shcore.dll Base Address: 0x7FF9539C0000 Size: 679936
msdmo.dll Base Address: 0x7FF93DF00000 Size: 16384
MFReadWrite.dll Base Address: 0x7FF91AD60000 Size: 995328
PROPSYS.dll Base Address: 0x7FF94BEA0000 Size: 688128
ResampleDmo.DLL Base Address: 0x7FF90D830000 Size: 503808
WTSAPI32.dll Base Address: 0x7FF94E4B0000 Size: 40960
d3d9.dll Base Address: 0x7FF946AF0000 Size: 1306624
D3DCOMPILER_43.dll Base Address: 0x7FF90EAC0000 Size: 2423296
DINPUT8.dll Base Address: 0x7FF920700000 Size: 176128
XINPUT1_3.dll Base Address: 0x400000 Size: 85504
SETUPAPI.dll Base Address: 0x7FF953540000 Size: 937984
bcrypt.dll Base Address: 0x7FF950950000 Size: 110592
IPHLPAPI.DLL Base Address: 0x7FF94FB00000 Size: 122880
powrprof.dll Base Address: 0x7FF950890000 Size: 77824
winmmbase.dll Base Address: 0x7FF923E80000 Size: 102400
cfgmgr32.dll Base Address: 0x7FF950C60000 Size: 221184
dwmapi.dll Base Address: 0x7FF94EA20000 Size: 90112
dxcore.dll Base Address: 0x7FF94E6F0000 Size: 131072
GFSDK_TXAA_AlphaResolve.win64.dll Base Address: 0x7FF94C260000 Size: 48128
GFSDK_ShadowLib.win64.dll Base Address: 0x7FF8EF4D0000 Size: 96256
OPENGL32.dll Base Address: 0x7FF917A60000 Size: 798720
GLU32.dll Base Address: 0x7FF947760000 Size: 126976
d3dx9_43.dll Base Address: 0x7FF8F9790000 Size: 2280960
bink2w64.dll Base Address: 0x7FF902890000 Size: 307712
MFCORE.DLL Base Address: 0x7FF8FAC70000 Size: 3252224
UMPDC.dll Base Address: 0x7FF9507F0000 Size: 36864
kernel.appcore.dll Base Address: 0x7FF950010000 Size: 36864
bcryptPrimitives.dll Base Address: 0x7FF951260000 Size: 380928
RTWorkQ.DLL Base Address: 0x7FF9300C0000 Size: 102400
MSASN1.dll Base Address: 0x7FF950C40000 Size: 36864
inputhost.dll Base Address: 0x7FF947490000 Size: 1515520
CoreMessaging.dll Base Address: 0x7FF94E020000 Size: 860160
CRYPTBASE.DLL Base Address: 0x7FF950750000 Size: 12288
sdk_legacy_led_x64.dll Base Address: 0x7FF8EDBA0000 Size: 2557952
VCRUNTIME140.dll Base Address: 0x7FF93A890000 Size: 65024
MSVCP140.dll Base Address: 0x7FF93A800000 Size: 346112
VCRUNTIME140_1.dll Base Address: 0x7FF93A8F0000 Size: 15360
CRYPTSP.dll Base Address: 0x7FF950730000 Size: 57344
rsaenh.dll Base Address: 0x7FF94FF80000 Size: 147456
mswsock.dll Base Address: 0x7FF950470000 Size: 331776
windows.storage.dll Base Address: 0x7FF94EFE0000 Size: 6909952
wintypes.dll Base Address: 0x7FF94EEA0000 Size: 544768
dbghelp.dll Base Address: 0x7FF9461A0000 Size: 1626112
dbgcore.DLL Base Address: 0x7FF939F30000 Size: 131072
uxtheme.dll Base Address: 0x7FF94E610000 Size: 425984
clbcatq.dll Base Address: 0x7FF952010000 Size: 495616
amsi.dll Base Address: 0x7FF9493E0000 Size: 57344
USERENV.dll Base Address: 0x7FF950560000 Size: 90112
profapi.dll Base Address: 0x7FF950EE0000 Size: 73728
MpOav.dll Base Address: 0x7FF946D30000 Size: 204800
cryptnet.dll Base Address: 0x7FF949BC0000 Size: 139264
drvstore.dll Base Address: 0x7FF949850000 Size: 1044480
devobj.dll Base Address: 0x7FF950CB0000 Size: 135168
wldp.dll Base Address: 0x7FF950810000 Size: 184320
imagehlp.dll Base Address: 0x7FF953420000 Size: 49152
nvspcap64.dll Base Address: 0x7FF900170000 Size: 2055680
ntmarta.dll Base Address: 0x7FF950030000 Size: 147456
EOSSDK-Win64-Shipping.dll Base Address: 0x7FF8ED180000 Size: 7229440
WLDAP32.dll Base Address: 0x7FF952520000 Size: 278528
Normaliz.dll Base Address: 0x7FF952230000 Size: 4096
WINHTTP.dll Base Address: 0x7FF94B6C0000 Size: 958464
Secur32.dll Base Address: 0x7FF932980000 Size: 12288
SSPICLI.DLL Base Address: 0x7FF950260000 Size: 126976
NSI.dll Base Address: 0x7FF951D30000 Size: 8192
dhcpcsvc6.DLL Base Address: 0x7FF94B640000 Size: 28672
dhcpcsvc.DLL Base Address: 0x7FF94B620000 Size: 69632
napinsp.dll Base Address: 0x7FF9379A0000 Size: 53248
pnrpnsp.dll Base Address: 0x7FF937020000 Size: 69632
DNSAPI.dll Base Address: 0x7FF94FBC0000 Size: 716800
winrnr.dll Base Address: 0x7FF933520000 Size: 32768
wshbth.dll Base Address: 0x7FF928450000 Size: 45056
nlansp_c.dll Base Address: 0x7FF931BB0000 Size: 81920
D3D11.DLL Base Address: 0x7FF94D030000 Size: 1867776
dxgi.dll Base Address: 0x7FF94E820000 Size: 651264
directxdatabasehelper.dll Base Address: 0x7FF94B5D0000 Size: 196608
nvapi64.dll Base Address: 0x7FF947940000 Size: 6148608
D3D10_1.DLL Base Address: 0x7FF902310000 Size: 147456
d3d10_1core.dll Base Address: 0x7FF947360000 Size: 12288
MSCTF.dll Base Address: 0x7FF951B30000 Size: 917504
textinputframework.dll Base Address: 0x7FF947790000 Size: 921600
CoreUIComponents.dll Base Address: 0x7FF94AEE0000 Size: 2036736
WINSTA.dll Base Address: 0x7FF9506C0000 Size: 229376
nvldumdx.dll Base Address: 0x7FF948160000 Size: 599552
rasadhlp.dll Base Address: 0x7FF949390000 Size: 8192
fwpuclnt.dll Base Address: 0x7FF94AE50000 Size: 294912
nvwgf2umx.dll Base Address: 0x7FF93EB70000 Size: 33575936
NvCamera64.dll Base Address: 0x7FF8EC930000 Size: 4086784
HID.DLL Base Address: 0x7FF94F8B0000 Size: 24576
XINPUT9_1_0.dll Base Address: 0x7FF94C280000 Size: 8192
WindowsCodecs.dll Base Address: 0x7FF94CB60000 Size: 1331200
d3dcompiler_47_64.dll Base Address: 0x7FF8EC520000 Size: 3922944
nvcuda64.dll Base Address: 0x7FF8E0320000 Size: 5634560
dcomp.dll Base Address: 0x7FF94D9B0000 Size: 1716224
MMDevApi.dll Base Address: 0x7FF9436F0000 Size: 462848
XAudio2_7.dll Base Address: 0x7FF8F3340000 Size: 485376
AUDIOSES.DLL Base Address: 0x7FF92A010000 Size: 1384448
resourcepolicyclient.dll Base Address: 0x7FF94EAA0000 Size: 36864
Windows.UI.dll Base Address: 0x7FF943FF0000 Size: 966656
avrt.dll Base Address: 0x7FF94B560000 Size: 8192
MessageBus.dll Base Address: 0x7FF936470000 Size: 5349888
gpapi.dll Base Address: 0x7FF950500000 Size: 77824
WINNSI.DLL Base Address: 0x7FF94E5A0000 Size: 16384
DPAPI.dll Base Address: 0x7FF950D10000 Size: 8192
socialclub.dll Base Address: 0x7FF8F49A0000 Size: 5077504
chrome_elf.dll Base Address: 0x7FF902350000 Size: 685568
libcef.dll Base Address: 0x7FF8D3B80000 Size: 109755392
COMDLG32.dll Base Address: 0x7FF953440000 Size: 774144
UIAutomationCore.DLL Base Address: 0x7FF91F080000 Size: 3047424
OLEACC.dll Base Address: 0x7FF947060000 Size: 274432
NETAPI32.dll Base Address: 0x7FF93A990000 Size: 49152
COMCTL32.dll Base Address: 0x7FF9445B0000 Size: 573440
DWrite.dll Base Address: 0x7FF94CDC0000 Size: 1605632
WINSPOOL.DRV Base Address: 0x7FF944DE0000 Size: 360448
USP10.dll Base Address: 0x7FF9468A0000 Size: 4096
dxva2.dll Base Address: 0x7FF93AB90000 Size: 102400
credui.dll Base Address: 0x7FF9264F0000 Size: 61440
urlmon.dll Base Address: 0x7FF942E50000 Size: 1298432
ncrypt.dll Base Address: 0x7FF950920000 Size: 114688
CRYPTUI.dll Base Address: 0x7FF933990000 Size: 278528
wevtapi.dll Base Address: 0x7FF94B660000 Size: 245760
ESENT.dll Base Address: 0x7FF935A90000 Size: 3112960
iertutil.dll Base Address: 0x7FF942B90000 Size: 557056
srvcli.dll Base Address: 0x7FF942B60000 Size: 57344
netutils.dll Base Address: 0x7FF94FAF0000 Size: 20480
WKSCLI.DLL Base Address: 0x7FF94B5B0000 Size: 49152
SAMCLI.DLL Base Address: 0x7FF943790000 Size: 61440
NTASN1.dll Base Address: 0x7FF9508E0000 Size: 69632
FirewallAPI.dll Base Address: 0x7FF94FA10000 Size: 380928
fwbase.dll Base Address: 0x7FF94F9A0000 Size: 163840
FWPolicyIOMgr.dll Base Address: 0x7FF93B380000 Size: 241664
wdmaud.drv Base Address: 0x7FF91FDB0000 Size: 200704
ksuser.dll Base Address: 0x7FF939EA0000 Size: 8192
msacm32.drv Base Address: 0x7FF9399B0000 Size: 20480
MSACM32.dll Base Address: 0x7FF92DFF0000 Size: 77824
midimap.dll Base Address: 0x7FF936090000 Size: 16384
ondemandconnroutehelper.dll Base Address: 0x7FF939CE0000 Size: 45056
TextShaping.dll Base Address: 0x7FF939C30000 Size: 315392
DiscordHook64.dll Base Address: 0x7FF8DF3F0000 Size: 762880
D3DCompiler_47.dll Base Address: 0x7FF94C770000 Size: 3043328
OneCoreCommonProxyStub.dll Base Address: 0x7FF944CC0000 Size: 8192
OneCoreUAPCommonProxyStub.dll Base Address: 0x7FF9485F0000 Size: 49152
YimMenu.dll Base Address: 0x7FF8C44E0000 Size: 5447168
schannel.DLL Base Address: 0x7FF94FE80000 Size: 544768
ncryptsslp.dll Base Address: 0x7FF9384E0000 Size: 86016
xinput1_4.dll Base Address: 0x7FF924F20000 Size: 32768
deviceaccess.dll Base Address: 0x7FF939400000 Size: 151552
Dumping registers:
RAX: 0x0
RCX: 0x2895F0C1C
RDX: 0x109FB8
RBX: 0x0
RSI: 0x7FF73E363A08
RDI: 0x122CF041071
RSP: 0xD5AAFFF8E0
RBP: 0x0
R8:  0xD5AAFFF890
R9:  0x0
R10: 0x2
R11: 0xD5AAFFF8D0
R12: 0x0
R13: 0x0
R14: 0x58C
R15: 0x0
Dumping stacktrace:
[0]     GTA5.exe+0x12E3AA8 0x7FF73E363AA8
[1]     GTA5.exe+0x12E93DA 0x7FF73E3693DA
[2]     KERNEL32.DLL BaseThreadInitThunk
[3]     ntdll.dll RtlUserThreadStart
--------End of exception--------
```

### Store

Epic Games

### Build

- [x] I've confirmed the issue exists on the latest version of YimMenu

**Comment by Yimura** [comment]
Does this happen every time you alt-tab the game?

**Comment by fludar** [comment]
yes

**Comment by fludar** [comment]
also the game

**Comment by Yimura** [comment]
Is this still an issue? Are you running the game in full-screen or windowed full-screen?

I run my game always in windowed full-screen and haven't had any issues and I've heard others say they don't have any issues running the game in normal full-screen either.

Try changing that and see if the issue goes away.

**Comment by Yimura** [comment]
Fixed by #1132
